src/templates/layout.html

```diff
@@ -34,6 +34,7 @@
     <li><a id="guides-claims" $if(active-guides-claims)$ class="active" $else$ $endif$ href="/guides/claims.html">Name Claims</a></li>
     <li><a id="guides-auctions" $if(active-guides-auctions)$ class="active" $else$ $endif$ href="/guides/auctions.html">Name Auctions</a></li>
     <li><a id="guides-wallet" $if(active-guides-wallet)$ class="active" $else$ $endif$ href="/guides/wallet.html">Wallet Setup</a></li>
+    <li><a id="guides-resources" $if(active-guides-resources)$ class="active" $else$ $endif$ href="/guides/resources.html">Handshake Resources</a></li>
```

Comment:
I feel like "resource" is a technical insider-term and could be confused with "resources" as in, like, guides and links, etc.

Maybe "HNS Record Types" or something direct like "register your name" or "add records to your name"?